api/serializers.py

```diff
@@ -31,3 +55,13 @@ class Meta:
             "feeders",
             "is_meta",
         )
+
+        read_only_fields = (
+            "id",
+            "hunt_id",
+            "sheet",
+            "answer",
+            "tags",
+            "metas",
+            "feeders",
+        )
```

Comment:
I'm not sure what the precise effects are of setting this (whether it will complain at deserialization or when saving the object or what) but yes the idea is to prevent update puzzle API calls from changing them. I'll loosen this as necessary when it comes up (though for tags I created a separate /tags endpoint for adding/removing them).